src/modules/theme/components/tag/tag.component.scss

```diff
@@ -0,0 +1,4 @@
+:host-context(.text-pre-wrap) strong {
```

Comment:
This is very not advisable to do. You can do it, but avoid if possible.

Nevertheless, approving for now...